README.md

```diff
@@ -30,10 +30,13 @@ Enter in the correct semantic version, and submit the release. This will create
 release with the provided version, as well as promote the CHANGELOG.md and upload the `.jar` 
 binary for clients to consume.
 
-### References
+## References
 * [Developer's Guide](https://www.metabase.com/docs/latest/developers-guide/start.html)
 * [Developer's Environment Guide](https://www.metabase.com/docs/latest/developers-guide/devenv.html)
 * [Driver Developer's Guide](https://www.metabase.com/docs/latest/developers-guide-drivers.html#driver-development)
 * [Driver Development Announcements](https://www.metabase.com/docs/latest/developers-guide-drivers.html#driver-development-announcements)
 * [Driver Development Form](https://discourse.metabase.com/c/driver-development)
 * [Metabase](https://www.metabase.com/)
+
+## Contact
+For feature requests and bugs, please create an issue in this repo. To contact support, please email: info@starburstdata.com
```

Comment:
we should refer to the starburst channel on Trino slack instead of an email. Also to the support page in the docs.starburst.io website maybe